CONTRIBUTING.md

```diff
@@ -2,17 +2,29 @@
 
 To make contributions to this charm, you'll need a working [development setup](https://juju.is/docs/sdk/dev-setup).
 
-You can create an environment for development with `tox`:
+This project uses `uv`. You can install it on Ubuntu with:
 
 ```shell
-tox devenv -e integration
-source venv/bin/activate
+sudo snap install --classic astral-uv --channel latest/edge
```

Comment:
Why install from edge

Reply:
It is not necessary anymore, when I started work on this, the support for PEP735 was only on edge. I moved back to stable.